### ISSUE TITLE
torch.linalg.cholesky warnings

### ISSUE THREAD

**Issue by ry-jojo** [joakimjohnander/DGPNet#4]
Hi Joakim, 
When training on 10shot, I am facing with warnings like the below:

   **_WARNING batched routines are designed for small sizes. It might be better to use the
   Native/Hybrid classical routines if you want good performance._**

I think this warning comes from the torch.linalg.cholesky(K_ss), when K_ss's largest size()  > 2048. 

May I know how you deal with this warning during training? Thanks in advance!

**Comment by joakimjohnander** [comment]
Hey,

Unfortunately, we have no simple fix that removes the warning. It seems to stem from a package, magma, that is called by PyTorch.

Best regards,
Joakim

**Comment by Parskatt** [comment]
@ry-jojo You can do a for-loop instead. Not sure about the performance loss/gains.
